internal/dsync/drwmutex.go

```diff
@@ -82,8 +82,8 @@ type Timeouts struct {
 // DefaultTimeouts contains default timeouts.
 var DefaultTimeouts = Timeouts{
 	Acquire:         drwMutexAcquireTimeout,
-	RefreshCall:     drwMutexUnlockCallTimeout,
-	UnlockCall:      drwMutexRefreshCallTimeout,
+	RefreshCall:     drwMutexRefreshCallTimeout,
```

Comment:
Okay so other than this change I think you should remove all other orthogonal changes that are in this PR.